src/components/Alert/Alert.js

```diff
@@ -69,11 +72,23 @@ export default class Alert extends React.Component {
       <div class={`
         chi-alert
         ${this.props.state && this.props.state !== 'base' ? `-${this.props.state}` : ''}
+        ${this.props.state && this.props.state !== 'base' ? `-b--${this.props.state}-light -bg--${this.props.state}-lighter` : ''}
+        ${this.props.state && this.props.state == 'inprogress' ? `-info` : ''}
+        ${this.props.state && this.props.state == 'inprogress' ? `-b--info-light -bg--info-lighter` : ''}
```

Comment:
```suggestion
```

Reply:
Looks like you use hardcoded `-info` color for `In progress` state. In this case In progress state will always render with a light blue style. As you can see in the screenshot, In progress spinner can also be present when the state is success, warning, danger, etc

I suggest making `In progress` a separate Prop (PropType boolean). It would just show / hide the spinner (no classname changes would be necessary as background color, text color, etc would be inherited from the main state class names)
<img width="730" alt="Screenshot 2021-05-17 at 13 02 08" src="https://user-images.githubusercontent.com/25136131/118478810-98738780-b710-11eb-9745-6c9ebbbed04c.png">